update_docs.py

```diff
@@ -0,0 +1,229 @@
+    'knights_archers_zombies' : { 'markdown': 'knights_archers_zombies.md', 'code' : 'pettingzoo/butterfly/knights_archers_zombies/knights_archers_zombies.py', 'environment' : (knights_archers_zombies_v7.env())},
+    'pistonball' : { 'markdown': 'pistonball.md', 'code' : 'pettingzoo/butterfly/pistonball/pistonball.py', 'environment' : (pistonball_v4.env())},
+    'prison' : { 'markdown': 'prison.md', 'code' : 'pettingzoo/butterfly/prison/prison.py', 'environment' : (prison_v3.env())},
+    'prospector' : { 'markdown': 'prospector.md', 'code' : 'pettingzoo/butterfly/prospector/prospector.py', 'environment' : (prospector_v4.env())},
+    'backgammon' : { 'markdown': 'backgammon.md', 'code' : 'pettingzoo/classic/backgammon/backgammon.py', 'environment' : (backgammon_v3.env())},
+    'checkers' : { 'markdown': 'checkers.md', 'code' : 'pettingzoo/classic/checkers/checkers.py', 'environment' : (checkers_v3.env())},
+    'chess' : { 'markdown': 'chess.md', 'code' : 'pettingzoo/classic/chess/chess_env.py', 'environment' : (chess_v5.env())},
+    'connect_four' : { 'markdown': 'connect_four.md', 'code' : 'pettingzoo/classic/connect_four/connect_four.py', 'environment' : (connect_four_v3.env())},
+    'dou_dizhu' : { 'markdown': 'dou_dizhu.md', 'code' : 'pettingzoo/rlcard_envs/rlcard_envs/dou_dizhu.py', 'environment' : (dou_dizhu_v4.env())},
+    'gin_rummy' : { 'markdown': 'gin_rummy.md', 'code' : 'pettingzoo/classic/rlcard_envs/gin_rummy.py', 'environment' : (gin_rummy_v4.env())},
+    'go' : { 'markdown': 'go.md', 'code' : 'pettingzoo/classic/go/go.py', 'environment' : (go_v5.env())},
+    'hanabi' : { 'markdown': 'hanabi.md', 'code' : 'pettingzoo/classic/hanabi/hanabi.py', 'environment' : (hanabi_v4.env())},
+    'leduc_holdem' : { 'markdown': 'leduc_holdem.md', 'code' : 'pettingzoo/classic/rlcard_envs/leduc_holdem.py', 'environment' : (leduc_holdem_v4.env())},
+    'mahjong' : { 'markdown': 'mahjong.md', 'code' : 'pettingzoo/classic/rlcard_envs/mahjong.py', 'environment' : (mahjong_v4.env())},
+    'rps' : { 'markdown': 'rps.md', 'code' : 'pettingzoo/classic/rps/rps.py', 'environment' : (rps_v2.env())},
+    'texas_holdem' : { 'markdown': 'texas_holdem.md', 'code' : 'pettingzoo/classic/rlcard_envs/texas_holdem.py', 'environment' : (texas_holdem_v4.env())},
+    'texas_holdem_no_limit' : { 'markdown': 'texas_holdem_no_limit.md', 'code' : 'pettingzoo/classic/rlcard_envs/texas_holdem_no_limit.py', 'environment' : (texas_holdem_no_limit_v6.env())},
+    'tictactoe' : { 'markdown': 'tictactoe.md', 'code' : 'pettingzoo/classic/tictactoe/tictactoe.py', 'environment' : (tictactoe_v3.env())},
+    'uno' : { 'markdown': 'uno.md', 'code' : 'pettingzoo/classic/rlcard_envs/uno.py', 'environment' : (uno_v4.env())},
+    'adversarial_pursuit' : { 'markdown': 'adversarial_pursuit.md', 'code' : 'pettingzoo/magent/adversarial_pursuit_v3.py', 'environment' : (adversarial_pursuit_v3.make_env())},
```

Comment:
This call to make_env is giving me an error when I try to run it. I assume this should just be `.env()`?